### ISSUE TITLE
The netnode for "$ tagcache" seems to be being reused between separate databases.

### ISSUE THREAD

**Issue by arizvisa** [arizvisa/ida-minsc#65]
The "$ tagcache" capability of ida-minsc is based around the contents of a blob started in a netnode. The idea is that the netnode is created when the database is created and then just referenced everytime it is needed. It seems that the logic for creating the netnode's identifier is scoped according to the `internal.comment.tagging` namespace as opposed to being scope according to the database itself. This netnode is instantiated by either the "idp.ev_init" or "idp.init" hooks.

The issue is because the identifier for this netnode is started within the class' definition and then only updated if the attribute is missing. This means that we're essentially trashing any blob that has the same netnode id when we open up another database which can also result in some weird values being displayed within the cache.

The entirety of the busted logic is in the implementation of the `tagging` base class within `base/_comment.py`:
```
    @classmethod
    def __init_tagcache__(cls, idp_modname):
        cls.node()
        logging.fatal(u"{:s}.init_tagcache('{:s}') : Initialized tagcache with netnode \"{:s}\" and node id {:#x}.".format('.'.join(('internal', __name__, cls.__name__)), internal.utils.string.escape(idp_modname, '\''), internal.utils.string.escape(cls.__node__, '"'), cls.__nodeid__))

    @classmethod
    def __nw_init_tagcache__(cls, nw_code, is_old_database):
        idp_modname = idaapi.get_idp_name()
        return cls.__init_tagcache__(idp_modname)

    @classmethod
    def node(cls):
        if hasattr(cls, '__nodeid__'):
            return cls.__nodeid__
        node = internal.netnode.get(cls.__node__)
        if node == idaapi.BADADDR:
            node = internal.netnode.new(cls.__node__)
        cls.__nodeid__ = node
        return node
```

**Comment by arizvisa** [comment]
This issue is fixed by PR #66.

**Comment by arizvisa** [comment]
Closing this issue as it was fixed by PR #66.
